crates/compiler/load/tests/test_reporting.rs

```diff
@@ -14701,6 +14706,54 @@ All branches in an `if` must have the same type!
         "###
     );
 
+    test_report!(
+        return_in_bare_statement,
```

Comment:
These are duplicated versions of `try`-based tests, meaning we now have separate tests for `try` and `return`